cityofaustin/atd-knack
# Update Data Tracker JS to prevent user from changing their assignment

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/9053

Lost track of this code review because it was challenging to find a time to demo to stakeholders. I'll do a better job getting code reviews earlier next time around. 👍 

## Testing
**URL to test:** 
_This has been tested and approved by Diana and stakeholders._

`view_1048`:
https://atd.knack.com/test--austin-transportation-data-tracker--31-jan-2022#my-work-orders/my-work-order-details2/61f852ef4373c6177acb3dd8/re-assign/61f852ef4373c6177acb3dd8/

`view_1146`:
https://atd.knack.com/test--austin-transportation-data-tracker--31-jan-2022#work-orders/work-order-details/6227c624c26314001f57d1a1/assign/6227c624c26314001f57d1a1/

**Steps to test:**
1. Go to one of the above links
2. Log in as the user `Test-Tech | AMD Data Tracker` stored in the Knack Shared 1Pass vault
3. Visit both links above and see that the **Lead Technician** field is disabled since the Lead Technician is set as Test Technician (the same user that you are logged in as). This prevents and assigned technician from re-assigning themselves.
